src/main/java/com/google/cloud/genomics/utils/Paginator.java

```diff
@@ -203,9 +204,9 @@ private Datasets(Genomics genomics) {
       super(genomics);
     }
 
-    @Override Genomics.Datasets.List createSearch(Genomics.Datasets api, Long request,
+    @Override Genomics.Datasets.List createSearch(Genomics.Datasets api, String request,
```

Comment:
I was referring to Paginator.java line 280. But looking closer at the code, the argument passed into operations.list isn't used for anything. How about something like this?

    @Override Genomics.Operations.List createSearch(Genomics.Operations api, 
          Optional<String> pageToken) throws IOException {		          Optional<String> pageToken) throws IOException {
       return api.search(pageToken		 +      return pageToken.isPresent() ? api.list(request).setPageToken(pageToken.get())
           .transform(		 +          : api.list("unused argument");